### PR TITLE
CheckBox: use Font Awesome plus/minus icons, remove unused images

### PR DESCRIPTION
**Pull request by xiphon**
Closes https://github.com/monero-project/monero-gui/issues/2488

### CODE REVIEW

**Comment by rating89us** [comment]
I'm not sure checked / unchecked should be used for pages other than the send page.

The + icon is only used as a collapse/expand function in the send page. In other pages it is used only as a common button.

File: pages/Account.qml [hunk 1]
```diff
@@ -349,8 +349,9 @@ Rectangle {
                 id: addNewAccountCheckbox 
                 visible: !selectAndSend
                 border: false
-                checkedIcon: "qrc:///images/plus-in-circle-medium-white.png" 
-                uncheckedIcon: "qrc:///images/plus-in-circle-medium-white.png" 
+                checkedIcon: FontAwesome.minusCircle
```
Comment (selsta) [inline pages/Account.qml]: 2019-11-25 06:31:38.540	W qrc:/pages/Account.qml:353: ReferenceError: FontAwesome is not defined

File: pages/Transfer.qml [hunk 2]
```diff
@@ -370,8 +371,9 @@ Rectangle {
               CheckBox {
                   id: paymentIdCheckbox
                   border: false
-                  checkedIcon: "qrc:///images/plus-in-circle-medium-white.png"
-                  uncheckedIcon: "qrc:///images/plus-in-circle-medium-white.png"
+                    checkedIcon: FontAwesome.minusCircle
```
Comment (selsta) [inline pages/Transfer.qml]: not indented

Comment (xiphon) [reply pages/Transfer.qml]: Actually is the correct indentation. The lines above and below are not indented properly.

Comment (selsta) [reply pages/Transfer.qml]: yep, half of the Transfer page is all wrong with indentation. If you prefer to use right indentation with new code also ok

Comment (xiphon) [reply pages/Transfer.qml]: > If you prefer to use right indentation with new code also ok

Yeah, kinda tired of manually adjusting the spaces

**Comment by xiphon** [comment]
> I'm not sure checked / unchecked should be used for the + icons in pages other than the send page.

Yep, fixed

**Comment by rating89us** [comment]
Can we reduce the space between + icon and Text? see below
image

**Comment by luigi1111** [comment]
@rating89us PR welcome or consider making a new issue regarding this. :)

**Pull request merged**